tests/Mapper/ORM/DoctrineORMMapperTest.php

```diff
@@ -80,4 +104,13 @@ public function testInheritanceTypes(): void
         $m->setAccessible(true);
         $m->invoke($mapper, $this->metadata);
     }
+
+    public function testDoctrineMappingLoaded(): void
```

Comment:
IMO this test should belong to a separated class inside Symfony as we are mixing here unit and functional tests.

Reply:
IMO this class does not have useful tests. What about removing the other tests and keep only this one? Adding the 2 missing test: discriminators and inheritanceType should cover the same as before but with better coverage and testing that actually this code works.

Reply:
That's another way of seeing it 🤣I didn't pay attention to the other tests, you're right 👍